Doesn't compile on gcc15

```
gcc -c rpc.c -o rpc.o -Wall -std=c99 -D_DEFAULT_SOURCE -Wno-missing-braces -Wno-unused-value -Wno-pointer-sign  -s -O2 -I. -Iexternal -I/usr/local/include/ -DPLATFORM_DESKTOP
In file included from rpc.c:113:
external/miniz.c:3089:9: note: ‘#pragma message: Using fopen, ftello, fseeko, stat() etc. path for file I/O - this path may not support large files.’
 3089 | #pragma message("Using fopen, ftello, fseeko, stat() etc. path for file I/O - this path may not support large files.")
      |         ^~~~~~~
rpc.c:120:9: warning: ‘RINI_MAX_VALUE_CAPACITY’ redefined
  120 | #define RINI_MAX_VALUE_CAPACITY     256
      |         ^~~~~~~~~~~~~~~~~~~~~~~
In file included from rpconfig.h:307,
                 from rpc.c:75:
external/rini.h:184:13: note: this is the location of the previous definition
  184 |     #define RINI_MAX_VALUE_CAPACITY         128
      |             ^~~~~~~~~~~~~~~~~~~~~~~
rpc.c:121:9: warning: ‘RINI_MAX_TEXT_SIZE’ redefined
  121 | #define RINI_MAX_TEXT_SIZE          256
      |         ^~~~~~~~~~~~~~~~~~
external/rini.h:176:13: note: this is the location of the previous definition
  176 |     #define RINI_MAX_TEXT_SIZE              128
      |             ^~~~~~~~~~~~~~~~~~
rpc.c:122:9: warning: ‘RINI_KEY_SPACING’ redefined
  122 | #define RINI_KEY_SPACING             37
      |         ^~~~~~~~~~~~~~~~
external/rini.h:190:13: note: this is the location of the previous definition
  190 |     #define RINI_KEY_SPACING                 36
      |             ^~~~~~~~~~~~~~~~
rpc.c:123:9: warning: ‘RINI_VALUE_SPACING’ redefined
  123 | #define RINI_VALUE_SPACING           35
      |         ^~~~~~~~~~~~~~~~~~
external/rini.h:195:13: note: this is the location of the previous definition
  195 |     #define RINI_VALUE_SPACING               32
      |             ^~~~~~~~~~~~~~~~~~
rpc.c: In function ‘ProcessCommandLine’:
rpc.c:1431:28: error: passing argument 1 of ‘strcpy’ from incompatible pointer type [-Wincompatible-pointer-types]
 1431 |                     strcpy(rpcFileName, argv[i + 1]);
      |                            ^~~~~~~~~~~
      |                            |
      |                            char **
In file included from rpconfig.h:309:
/usr/include/string.h:141:39: note: expected ‘char * restrict’ but argument is of type ‘char **’
  141 | extern char *strcpy (char *__restrict __dest, const char *__restrict __src)
      |                      ~~~~~~~~~~~~~~~~~^~~~~~
rpc.c:1526:24: warning: comparison between pointer and zero character constant [-Wpointer-compare]
 1526 |     if (rpcFileName[0] != '\0')
      |                        ^~
rpc.c:1526:9: note: did you mean to dereference the pointer?
 1526 |     if (rpcFileName[0] != '\0')
      |         ^
rpc.c:1529:56: error: passing argument 1 of ‘LoadProjectConfigRaw’ from incompatible pointer type [-Wincompatible-pointer-types]
 1529 |         rpcProjectConfigRaw raw = LoadProjectConfigRaw(rpcFileName);
      |                                                        ^~~~~~~~~~~
      |                                                        |
      |                                                        char **
rpconfig.h:313:54: note: expected ‘const char *’ but argument is of type ‘char **’
  313 | rpcProjectConfigRaw LoadProjectConfigRaw(const char *fileName)
      |                                          ~~~~~~~~~~~~^~~~~~~~
rpc.c: At top level:
rpc.c:2076:13: warning: ‘UnpackDirectoryData’ defined but not used [-Wunused-function]
 2076 | static void UnpackDirectoryData(const char *outputDirPath, const unsigned char *data, int *dataSize, PackFileEntry *entries, int fileCount)
      |             ^~~~~~~~~~~~~~~~~~~
rpc.c:2020:14: warning: ‘PackDirectoryData’ defined but not used [-Wunused-function]
 2020 | static char *PackDirectoryData(const char *baseDirPath, int *packSize)
      |              ^~~~~~~~~~~~~~~~~
rpc.c:292:12: warning: ‘exportFormatActive’ defined but not used [-Wunused-variable]
  292 | static int exportFormatActive = 0;         // ComboBox file type selection
      |            ^~~~~~~~~~~~~~~~~~
rpc.c:261:12: warning: ‘supportMessageRandBtn’ defined but not used [-Wunused-variable]
  261 | static int supportMessageRandBtn = 0;       // Support message buttons random position
      |            ^~~~~~~~~~~~~~~~~~~~~
rpc.c:259:13: warning: ‘showSupportMessage’ defined but not used [-Wunused-variable]
  259 | static bool showSupportMessage = false;
      |             ^~~~~~~~~~~~~~~~~~
rpc.c:208:15: warning: ‘currentTime’ defined but not used [-Wunused-variable]
  208 | static double currentTime = 0;              // Current time counter in seconds
      |               ^~~~~~~~~~~
rpc.c:207:15: warning: ‘baseTime’ defined but not used [-Wunused-variable]
  207 | static double baseTime = 0;                 // Base time in seconds to start counting
      |               ^~~~~~~~
rpc.c:204:13: warning: ‘showMessageReset’ defined but not used [-Wunused-variable]
  204 | static bool showMessageReset = false;       // Show message: reset
      |             ^~~~~~~~~~~~~~~~
rpc.c:191:18: warning: ‘panelView’ defined but not used [-Wunused-variable]
  191 | static Rectangle panelView = { 0 };         // Project properties panel view (for scissoring)
      |                  ^~~~~~~~~
rpc.c:184:16: warning: ‘mousePoint’ defined but not used [-Wunused-variable]
  184 | static Vector2 mousePoint = { 0 };          // Mouse position
      |                ^~~~~~~~~~
rpc.c:183:12: warning: ‘framesCounter’ defined but not used [-Wunused-variable]
  183 | static int framesCounter = 0;               // General pourpose frames counter (not used)
      |            ^~~~~~~~~~~~~
make[1]: *** [Makefile:330: rpc.o] Error 1
make[1]: Leaving directory '/home/ykpcx/raylib-project-creator/src'
make: *** [Makefile:321: all] Error 2
```

CC=clang works despite the warning

--- a/src/rpc.c
+++ b/src/rpc.c
@@ -1391,7 +1391,7 @@ static void ProcessCommandLine(int argc, char *argv[])
 
     if (argc == 1) showUsageInfo = true;
 
-    char *rpcFileName[256] = { 0 }; // Provided input rpc config file, overwrites any other property
+    char rpcFileName[256] = { 0 }; // Provided input rpc config file, overwrites any other property
     rpcProjectConfig *config = (rpcProjectConfig *)RL_CALLOC(1, sizeof(rpcProjectConfig));
 
     // Process command line arguments

